addon/components/frost-textarea.js

```diff
@@ -48,6 +49,7 @@ export default Component.extend(FrostEventsProxyMixin, {
       isClearEnabled: false,
       isClearVisible: false,
       disabled: false,
+      onClear: () => {},
```

Comment:
This does not match the pattern used elsewhere in the repo, such as at https://github.com/ciena-frost/ember-frost-core/blob/master/addon/components/frost-expand.js#L94

Was this deviation intentional?